zcash_primitives/src/transaction/components/sapling.rs

```diff
@@ -255,9 +255,10 @@ impl OutputDescription<GrothProofBytes> {
         let cmu = read_base(&mut reader, "cmu")?;
 
         // Consensus rules (§4.5):
-        // - Canonical encoding is enforced here.
+        // - Canonical encoding is enforced in librustzcash_sapling_check_output by zcashd
```

Comment:
Are there any clients other than zcashd for which this change would be unsafe?

Reply:
- `zcashd` enforces this as documented, so this will continue to be enforced for all mined notes.
- `zebrad` (I believe) does its own thing, so shouldn't be affected by this change.
- `ephemeral_key` is always parsed (and thus validated) as part of trial-decryption (for light clients in particular), so this will continue to be enforced for individual users' notes.